src/components/PrimaryPanes/index.js

```diff
@@ -135,6 +134,8 @@ class PrimaryPanes extends Component {
   }
 }
 
+PrimaryPanes.displayName = "PrimaryPanes";
```

Comment:
Small nit: we no longer need displaynames

Reply:
Corrected